tests/Unit/Elliptic/Systems/Elasticity/Test_Equations.cpp

```diff
@@ -0,0 +1,126 @@
+      &primal_fluxes<Dim>, "Equations",
+      {MakeString{} << "primal_fluxes_wrapper_" << Dim << "d"},
+      {{{-1., 1.}, {-1., 1.}, {0., 1.}, {0., 1.}}}, used_for_size);
+  pypp::check_with_random_values<1>(
+      &Elasticity::auxiliary_fluxes<Dim>, "Equations",
+      {MakeString{} << "auxiliary_fluxes_" << Dim << "d"}, {{{-1., 1.}}},
+      used_for_size);
+}
+
+template <size_t Dim>
+void test_computers(const DataVector& used_for_size) {
+  using field_tag = Elasticity::Tags::Displacement<Dim>;
+  using auxiliary_field_tag = Elasticity::Tags::Strain<Dim>;
+  using field_flux_tag =
+      ::Tags::Flux<field_tag, tmpl::size_t<Dim>, Frame::Inertial>;
+  using auxiliary_flux_tag =
+      ::Tags::Flux<auxiliary_field_tag, tmpl::size_t<Dim>, Frame::Inertial>;
+  using field_source_tag = ::Tags::Source<field_tag>;
+  using constitutive_relation_tag = Elasticity::Tags::ConstitutiveRelation<
+      Elasticity::ConstitutiveRelations::IsotropicHomogeneous<Dim>>;
+  using coordinates_tag = Tags::Coordinates<Dim, Frame::Inertial>;
+  const size_t num_points = used_for_size.size();
+  {
+    INFO("Fluxes" << Dim << "D");
+    auto box = db::create<db::AddSimpleTags<
+        field_tag, auxiliary_field_tag, field_flux_tag, auxiliary_flux_tag,
+        constitutive_relation_tag, coordinates_tag>>(
+        tnsr::I<DataVector, Dim>{num_points, 1.},
+        tnsr::ii<DataVector, Dim>{num_points, 2.},
+        tnsr::IJ<DataVector, Dim>{num_points, 3.},
+        tnsr::Ijj<DataVector, Dim>{num_points, 4.},
+        Elasticity::ConstitutiveRelations::IsotropicHomogeneous<Dim>{1., 1.},
+        tnsr::I<DataVector, Dim>{num_points, 6.});
+
+    const Elasticity::Fluxes<Dim> fluxes_computer{};
+    using argument_tags = typename Elasticity::Fluxes<Dim>::argument_tags;
+
+    db::mutate_apply<tmpl::list<field_flux_tag>, argument_tags>(
+        fluxes_computer, make_not_null(&box), get<auxiliary_field_tag>(box));
+    auto expected_field_flux = tnsr::IJ<DataVector, Dim>{num_points, 3.};
+    Elasticity::primal_fluxes(make_not_null(&expected_field_flux),
+                              get<auxiliary_field_tag>(box),
+                              get<constitutive_relation_tag>(box),
+                              get<coordinates_tag>(box));
+    CHECK(get<field_flux_tag>(box) == expected_field_flux);
+
+    db::mutate_apply<tmpl::list<auxiliary_flux_tag>, argument_tags>(
+        fluxes_computer, make_not_null(&box), get<field_tag>(box));
+    auto expected_auxiliary_flux = tnsr::Ijj<DataVector, Dim>{num_points, 4.};
+    Elasticity::auxiliary_fluxes(make_not_null(&expected_auxiliary_flux),
+                                 get<field_tag>(box));
+    CHECK(get<auxiliary_flux_tag>(box) == expected_auxiliary_flux);
+  }
+  {
+    INFO("Sources" << Dim << "D");
+    auto box = db::create<db::AddSimpleTags<field_tag, field_source_tag>>(
+        tnsr::I<DataVector, Dim>{num_points,
+                                 std::numeric_limits<double>::signaling_NaN()},
```

Comment:
you can remove the `field_tag` from this DataBox since it's not needed to compute sources

Reply:
I tried removing it, but it does seem to be necessary.

Reply:
ok, looks like the interface wants it but it's unused. let's keep it then :)